src/book/Chapter3-Proofs.lagda.md

```diff
@@ -1178,7 +1178,7 @@ dominoes. This property of equality---that we're allowed to such a thing in the
 first place---is called *transitivity,* and we can be stated as:
 
 ```agda
-  trans&
```

Comment:
This should probably be `trans⅋`

Reply:
Thanks for clarification. I just noticed the superfluous "&" in the PDF and deleted it in the source :smile: